skycoin/skywire
# Add cli command visor ping and test

Did you run `make format && make check`?

Fixes #1377 

 Changes:	
- [x] add `ping` command as visor subcommand in `skywire-cli visor ping <pk>`
- [x] add `test` command as visor subcommand in `skywire-cli visor test`

How to test this PR:
**PING**
1. Start a visor A
2. Start visor B
3. On the machine with visor B run command `skywire-cli visor ping <visor-a-pk> -t <tries> -s <size-of-packet>`

**TEST**
1. Run a public visor
2. Start a visor, then run test command by `skywire-cli visor test -t <tries> -s <size-of-packet>`

Result should be like this, but number of test should be same as number of public visors on service discovery
![Screenshot_2022-12-21_02-20-51](https://user-images.githubusercontent.com/79150699/208782070-70cd0554-187d-4092-b230-8596e50813e4.jpg)
